### PR TITLE
Decouple `editTargetPort` from `createContainerApp`

### PR DESCRIPTION
**Pull request by MicroFish91**

### CODE REVIEW

File: src/commands/createContainerApp/EnableIngressStep.ts [hunk 1]
```diff
@@ -23,7 +23,8 @@ export class EnableIngressStep extends AzureWizardPromptStep<ICreateContainerApp
     public async getSubWizard(context: ICreateContainerAppContext): Promise<IWizardOptions<ICreateContainerAppContext> | undefined> {
         if (context.enableIngress) {
             return {
-                promptSteps: [new IngressVisibilityStep(), new TargetPortStep()]
+                // @ts-ignore
```
Comment (MicroFish91) [inline src/commands/createContainerApp/EnableIngressStep.ts:26]: The context types are temporarily incompatible until all the ingress commands are decoupled, just doing `@ts-ignore` for now, will remove when `enableIngress` is migrated

File: src/commands/ingress/IngressUpdateBaseStep.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+import { IContainerAppContext } from "../IContainerAppContext";
+import { updateContainerApp } from "../deployContainerApp/updateContainerApp";
+
+type IngressOptions = {
+    ingress: Ingress | null,
+    working: string,
+    workCompleted: string
+}
+
+export abstract class IngressUpdateBaseStep<T extends IContainerAppContext> extends AzureWizardExecuteStep<T> {
```
Comment (MicroFish91) [inline src/commands/ingress/IngressUpdateBaseStep.ts:19]: This step is meant to eventually completely replace the use of `updateIngressSettings.ts`

**Comment by MicroFish91** [comment]
`ICreateContainerAppContext` should eventually extend `IngressContext`, however, there is a temporary incompatibility because not all the ingress commands have been wired up to support the activity log.  This will change by the last of the ingress update PRs.

File: src/commands/createContainerApp/EnableIngressStep.ts (hunk 1 above)
Comment (bwateratmsft) [reply src/commands/createContainerApp/EnableIngressStep.ts:26]: Maybe add a todo comment and that explanation?

**Review by bwateratmsft — APPROVED** [review]

**Pull request merged**